ert3/console/_console.py

```diff
@@ -78,6 +96,44 @@ def _build_argparser():
     return parser
 
 
+def _get_ert3_examples_path():
+    pkg_examples_path = pathlib.Path(pkg.resource_filename("ert3_examples", ""))
+    # check that examples folder exist
+    if not pkg_examples_path.exists():
+        raise ert3.exceptions.IllegalWorkspaceOperation(
+            "Examples folder was not found."
+        )
+    return pkg_examples_path
+
+
+def _init(args):
+    assert args.sub_cmd == "init"
+
+    if args.example is None:
+        ert3.workspace.initialize(pathlib.Path.cwd())
+    else:
+        example_name = args.example
+        pkg_examples_path = _get_ert3_examples_path()
+        pkg_example_path = pkg_examples_path / example_name
+        wd_example_path = pathlib.Path.cwd() / example_name
+
+        # check that examples folder contains provided 'example_name'
+        if not pkg_example_path.exists():
+            raise ert3.exceptions.IllegalWorkspaceOperation(
+                f"Example {example_name} is not a valid ert3 example."
```

Comment:
Should we list the available examples? If moved into a function as suggested above, it should be quite easy to append the available examples 👍🏻